src/store/traction/saphyr/requests/getters.js

```diff
@@ -1,6 +1,10 @@
 const getters = {
-  requests: (state) => state.requests,
-  requestsRequest: (state, getters, rootState) => rootState.api.v1.traction.saphyr.requests,
+  requests: (state) => {
+    return Object.values(state.requests)
+  },
+  requestsRequest: (state, getters, rootState) => {
```

Comment:
This has always vexed me. It might just be better to use this directly in place but because this is Saphyr and it is deprecated I am happy to leave as is.

Reply:
Yes, I thought about it as well and then it as it is Saphyr, thought it shouldn't be bothered.